fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -111,3 +111,232 @@ const getSimBriefOfp = (mcdu, updateView, callback = () => {}) => {
             updateView();
         });
 };
+
+/**
+ * There are two uplink requests that are made at the same time:
+ * - AOC ACT F-PLN
+ * - PERF DATA
+ */
+const insertUplink = (mcdu) => {
```

Comment:
This code seems to be from a bad merge or something. As that code has now all be relocated to typescript.
I think you will have to redo your changes on a branch that is clean from master. 

Simbrief uplink is now all happening here: https://github.com/flybywiresim/aircraft/blob/2cdf866fb140b72151c25e2413dd5365364ba08f/fbw-a32nx/src/systems/fmgc/src/flightplanning/new/uplink/SimBriefUplinkAdapter.ts#L115